tools/generator/CodeGeneratorOptions.cs

```diff
@@ -105,8 +106,9 @@ public static CodeGeneratorOptions Parse (string[] args)
 					"SDK Platform {VERSION}/API level.",
 					v => opts.ApiLevel = v },
 				{ "lang-features=",
-					"For internal use. (Flags: interface-constants,default-interface-methods,nested-interface-types,nullable-reference-types,obsoleted-platform-attributes,restrict-to-attributes,do-not-fix-obsolete-overrides)",
+					"For internal use. (Flags: interface-constants,default-interface-methods,nested-interface-types,nullable-reference-types,obsoleted-platform-attributes,restrict-to-attributes,do-not-fix-obsolete-overrides,emit-legacy-interface-invokers)",
 					v => {
+						opts.EmitLegacyInterfaceInvokers = v?.Contains ("emit-legacy-interface-invokers") == true;
```

Comment:
What is the intention with this flag? Will we use it internally? Is it an escape hatch if there are issues with this implementation? Are we going to publicly expose it for users?